copyright

```diff
@@ -1426,10 +1420,12 @@ Files:
  images/outfit/nuke*
  images/outfit/plasma?core*
  images/outfit/red?sun*
+ images/outfit/small?photovoltaic*
```

Comment:
These should be on a separate entry, they aren't only derived from MZ

Reply:
Oh I see. Right, I'll fix it right away.